src/redux/users/middleware.ts

```diff
@@ -0,0 +1,64 @@
+import { RedirectionProps } from '@allTypes/reduxTypes/viewsStateTypes'
+import API from '@axios/API'
+import { ILoginListParams } from '@axios/authentication/login/managerLoginTypes'
+import { IRegisterListParams } from '@axios/authentication/register/registerLoginTypes'
+import { AppDispatch } from '@redux/store'
+
+import slice from '../slices/views/slice'
+
+import user from './user'
+
+const { setIsLoading, setError, setSuccessfully } = user.actions
+
+const { setRedirection } = slice.actions
+
+const setRedirectionState = (value: RedirectionProps) => (dispatch: AppDispatch) => {
+  dispatch(setRedirection(value))
+}
+
+const login = (params: ILoginListParams) => async (dispatch: AppDispatch) => {
+  try {
+    dispatch(setIsLoading(true))
+
+    const response = await API.authLogin.login(params)
+
+    localStorage.setItem('accessToken', response.data.data.accessToken)
+    dispatch(setError(false))
+
+    if (response.status === 200) {
+      dispatch(setSuccessfully(true))
```

Comment:
We need to store user data in redux, if it's there then request is successfull so no need of setSuccessfully

Reply:
Fixed